tools/loader/Loader.cpp

```diff
@@ -267,7 +267,7 @@ void Loader::compile(Context &ctx) {
     // generateNodeQuantizationInfos() when writing out the profile, allowing
     // for both lowered and unlowered NodeValues to find their quantization
     // parameters.
-    ::lower(F_, &loweredMap_, /* backend */ nullptr,
+    ::lower(F_, &loweredMap_, /* backend */ nullptr, /* backendKind */ nullptr,
```

Comment:
Why do you need both the backend and the backend kind? Can you get rid of one of the parameters?